backend/layers/business/business.py

```diff
@@ -413,6 +414,10 @@ def remove_dataset_version(
         Removes a dataset version from an existing collection version
         """
         self._assert_collection_version_unpublished(collection_version_id)
+        dv = self.database_provider.get_dataset_version(dataset_version_id)
+        if not CorporaConfig().is_cxg_admin and dv.canonical_dataset.published_at:
+            raise CollectionUpdateException from None
```

Comment:
`is_cxg_admin` doesn't belong to the business logic, since it's user based. You should add this as a parameter and move this call to the API layer.

Reply:
It determines rules for the business logic; it is involved in API calls.

Reply:
Who will even set `is_cxg_admin`?

Reply:
Sorry, I should have included the link to the previous issue / PR #5194

Reply:
My thinking on the parameter vs env var: because the entire suite of interactions with the entire service code will either be _entirely_ cxg admin or _entirely_ not cxg admin, it makes more sense to have the logic served by an env var. 

A parameter such as 'get_tombstoned' doesn't make sense as an env var because it could (and does) change during a set of actions. The service, when running in deployed environments, will always have a value of `False` for `is_cxg_admin`.

Reply:
Also the business API should not expose the ability to perform cxg admin functions to the API layer, because cxg admin ops will never occur via the API.

Reply:
In this case, I'd just delete the method and move it to cxg_admin, or make it private. I think the idea of using an environment variable here is incredibly confusing.

Reply:
The method is still used for deleting unpublished data.

Reply:
Looking forward: as cxg admin scripts expand in scope, we run the risk of duplicating logic (risk of bugs) if we have to have separate functions for cxg admin work.